src/coreclr/jit/llvm.cpp

```diff
@@ -92,12 +92,16 @@ Llvm::Llvm(Compiler* compiler)
     : _compiler(compiler),
     _info(compiler->info),
     _function(nullptr),
+    _sigInfo(compiler->info.compMethodInfo->args),
     _builder(_llvmContext),
     _prologBuilder(_llvmContext),
+    _blkToLlvmBlkVectorMap(compiler->getAllocator(CMK_Codegen)),
+    _sdsuMap(compiler->getAllocator(CMK_Codegen)),
+    _localsMap(compiler->getAllocator(CMK_Codegen)),
+    _debugMetadataMap(compiler->getAllocator(CMK_Codegen)),
```

Comment:
@yowl I wanted to ask - what is the point of this map? We use it in `getOrCreateDebugMetadata`, but only once per compilation, so it should always go the "lookup failed" path (of which I am not sure what the consequences are).

Reply:
I think I was basically just copying

https://github.com/dotnet/runtimelab/blob/9c53250461c337d6febcdc2f43ea9619123b8354/src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/LLVMCodegenCompilation.cs#L59

Where it asks for the debug compilation unit for every IL instruction.